Dockerfile

```diff
@@ -1,38 +1,14 @@
-FROM ubuntu:14.04
-MAINTAINER Colin Alston <colin@praekelt.com>
-RUN apt-get update && apt-get -y --force-yes install libjpeg-dev zlib1g-dev libxslt1-dev libpq-dev nginx redis-server supervisor python-dev python-pip
-RUN apt-get -y install libffi-dev gettext
+FROM praekeltfoundation/django-bootstrap:onbuild
+RUN apt-get-install.sh git libjpeg-dev zlib1g-dev libffi-dev gettext libtiff-dev
```

Comment:
It's better not to install `-dev` packages because they'll pull in a lot of development tools and inflate the image size. I'm guessing some of these are needed for Pillow? Can you try install just the runtime libraries? I also try to add packages in an alphabetical list (but that's just me being fussy). Something like:
```dockerfile
RUN apt-get-install.sh \
    gettext \
    libjpeg62 \
    libtiff5
```
Do we need `git`?